SECP-service/src/main/resources/assets/app/views/error/404.html

```diff
@@ -0,0 +1,6 @@
+<link href="css/404.css" rel="stylesheet">
+<div>
+    <a href="http://www.secp.co"><img src="img/secp-logo.png"/></a>
```

Comment:
this should not point to secp.co. It should just point to "/". For example, if i am running localhost:8080/hfhfh. clicking on this takes me to secp.co